appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyEnvironment.java

```diff
@@ -207,7 +205,16 @@
     APPSERVER_TASK_BNS(
         "X-AppEngine-Appserver-Task-Bns",
         "com.google.appengine.runtime.appserver_task_bns",
-        "", true);
+        "", true),
+    HTTPS(
```

Comment:
Right.... But how is setting them? I am not sure they are set in the incoming request sent from the app server bridge (outside the jvm?)